rhscl-postgresql-apb/roles/rhscl-postgresql-apb-openshift/defaults/main.yml

```diff
@@ -1,4 +1,7 @@
 ---
 postgresql_password: "{{ lookup('password', '/tmp/pass chars=ascii_letters,digits') }}"
+postgresql_user: "admin"
```

Comment:
hmm, I'm not sure I like that. Any defaults you specify in the apb.yml would overwrite these, and leaving the defaults here makes testing a lot easier/faster since you don't need to punch in every variable when you're running the playbook by hand. I would consider a role that doesn't have a good defaults.yml incomplete.